django/contrib/auth/models.py

```diff
@@ -361,6 +360,11 @@ def email_user(self, subject, message, from_email=None, **kwargs):
         """
         send_mail(subject, message, from_email, [self.email], **kwargs)
 
+    def clean(self):
```

Comment:
I guess we should document this method in docs/topics/auth/customizing.txt and the release notes to advise anyone implementing a custom user model with a `clean()` method to call `super()`.